README.md

```diff
@@ -5,6 +5,8 @@ cf\_units
 
 \(C) British Crown Copyright 2015 - 2016, Met Office
 
+Introduction
+------------
```

Comment:
And yet the headings are all rendered correctly as top level or first sub-level headings...

Also, I quite like the look of the horizontal rules under each sub-heading.